integration_tests/steps/myco.py

```diff
@@ -32,6 +32,7 @@ def step_impl(context):
     sleep(5)
     context.matcher = TestRedisMycoMatcher()
     sleep(3)
+    context.matcher.request_area_id_name_map()
```

Comment:
if possible move this inside the __init method of the `TestRedisMycoMatcher` or inside the market cycle handler.
Furthermore, in the on_area_map handler of the test class, put `self.called_events.add("on_area_map")` there and inside the `all events handler are called` step it should become 
```python
assert context.matcher.called_events == {"market_cycle", "tick", "offers_bids_response",
                                             "event_or_response", "match", "finish", "area_map}
```